OJP/OJP_JourneySupport.xsd

```diff
@@ -353,6 +353,16 @@
 					<xs:documentation>Alighting will not be allowed at this stop of this journey.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:choice>
+				<xs:sequence>
+					<xs:element ref="siri:ExpectedDepartureOccupancy" minOccurs="0" maxOccurs="unbounded"/>
+					<xs:element ref="siri:ExpectedDepartureCapacities" minOccurs="0" maxOccurs="unbounded"/>
+				</xs:sequence>
+				<xs:sequence>
+					<xs:element ref="siri:RecordedDepartureOccupancy" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
What does "recorded" means for a journey planner ?... the answer is always expected to be about journeys is in some future, isn't it ?